dietpi/func/dietpi-logclear

```diff
@@ -142,6 +143,9 @@
 			((INFO_FILES_PROCESSED++))
 		done
 
+		# Truncate files that failed with bash '>' redirect
+		truncate -c -s0 ${ARRAY_LOGFILE_TRUNCATE_DENIED[*]}
```

Comment:
Should probably add a `[ ${#key[@]} -gt 0 ] && truncate ...` ?

Reply:
Makes sense. Also calling the array as array and quoting the elements, in case there are special characters or spaces within the path/file names:
```suggestion
		(( ${#ARRAY_LOGFILE_TRUNCATE_DENIED[@]} )) && truncate -c -s0 "${ARRAY_LOGFILE_TRUNCATE_DENIED[@]}"
```